### DISCUSSION TITLE
Build images without Azure

### DISCUSSION THREAD

**Discussion by brianjmurrell** [actions/virtual-environments#6502] [General questions]
I wonder if anyone has created a fork this repo with the goal of building these actions runner-images without Azure.

**Comment by gbakeman** [comment]
The nektos/act project (utility to run GitHub Workflows locally) has [a repository](https://github.com/nektos/act-environments) that's essentially forked this repository, but modified the Packer scripts to create Ubuntu images for a local docker instance. It's not clear to me how this would work for the windows images.
